### PR TITLE
As a developer I want to strip out the old login system (and owner from user table)

### PR DESCRIPTION
**Pull request by Ben-Edwards-cgi**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

**Comment by akikrahman1** [comment]
Have approved in principle as long as FE integration is done, the PR can be merged together

**Review by akikrahman1 — APPROVED** [review]
Need to make sure FE integration is done before merging into master

File: src/main/resources/db/migration/V1_90__temp_users.sql
Comment (npv0) [inline src/main/resources/db/migration/V1_90__temp_users.sql:1]: do not merge

**Review by npv0 — CHANGES_REQUESTED** [review]

**Pull request merged**
